src/napari/layers/points/points.py

```diff
@@ -2002,52 +2002,126 @@ def add(self, coords):
+                    indices,
                 ),
                 vertex_indices=((),),
             )
-            self._shown = np.delete(self._shown, index, axis=0)
-            self._size = np.delete(self._size, index, axis=0)
-            self._symbol = np.delete(self._symbol, index, axis=0)
-            self._border_width = np.delete(self._border_width, index, axis=0)
+            self._shown = np.delete(self._shown, indices, axis=0)
+            self._size = np.delete(self._size, indices, axis=0)
+            self._symbol = np.delete(self._symbol, indices, axis=0)
+            self._border_width = np.delete(self._border_width, indices, axis=0)
             with self._border.events.blocker_all():
-                self._border._remove(indices_to_remove=index)
+                self._border._remove(indices_to_remove=indices)
             with self._face.events.blocker_all():
-                self._face._remove(indices_to_remove=index)
-            self._feature_table.remove(index)
-            self.text.remove(index)
-            if self._value in self.selected_data:
+                self._face._remove(indices_to_remove=indices)
+            self._feature_table.remove(indices)
+            self.text.remove(indices)
+            if self._value in indices:
                 self._value = None
             else:
                 if self._value is not None:
                     # update the index of self._value to account for the
                     # data being removed
-                    indices_removed = np.array(index) < self._value
+                    indices_removed = np.array(indices) < self._value
                     offset = np.sum(indices_removed)
                     self._value -= offset
                     self._value_stored -= offset
 
-            self._set_data(np.delete(self.data, index, axis=0))
+            self._set_data(np.delete(self.data, indices, axis=0))
             self.events.data(
                 value=self.data,
                 action=ActionType.REMOVED,
                 data_indices=tuple(
-                    self.selected_data,
+                    indices,
                 ),
                 vertex_indices=((),),
             )
-            self.selected_data = set()
             self.events.features()
 
+    def remove_selected(self) -> None:
+        """Remove all selected points."""
+        self.remove(list(self.selected_data))
+        self.selected_data = set()
+
+    def get_point_info(self, index: int) -> dict:
+        """
+        Retrieve all available information about a point at the given index.
+
+        Parameters
+        ----------
+        index : int
+            Index of the point.
+
+        Returns
+        -------
+        dict
+            A dictionary containing all relevant details of the point.
+        """
+        if not (0 <= index < len(self.data)):
+            return {
+                'data': None,
+                'properties': {},
```

Comment:
Should we leave out properties since it's replaced byu features? Are we still trying to keep it around for some transition reason? @andy-sweet Do you remember the status of this?

Reply:
Pretty sure this indeed shouldn't return properties, so I changed it :)